src/Agent.Worker/Worker.cs

```diff
@@ -15,57 +16,7 @@ public interface IWorker : IAgentService
 
     public sealed class Worker : AgentService, IWorker
     {
-        private readonly TimeSpan WorkerStartTimeout = TimeSpan.FromSeconds(30);
-
-        private void InitializeSecrets(JobRequestMessage message)
```

Comment:
I moved this private method down. When reading the file, it's less important than the public method.